brayns/parameters/GeometryParameters.cpp

```diff
@@ -194,7 +204,11 @@ GeometryParameters::GeometryParameters()
         //
         (PARAM_CIRCUIT_MESH_TRANSFORMATION.c_str(),
          po::bool_switch()->default_value(false),
-         "Enable mesh transformation according to circuit information.");
+         "Enable mesh transformation according to circuit information.")
+        //
+        (PARAM_DEFAULT_BVH_FLAG.c_str(),
+         po::value<std::vector<std::string>>()->multitoken(),
+         "Default bvh type [static|dynamic|compact|robust]");
```

Comment:
It's not clear that you can have multiple flags and what is the token separator.

Reply:
BVH